backend/routes.js

```diff
@@ -218,6 +220,8 @@ const setup = ({
     amendCaseNNoteRouter({ elite2Api, caseNotesApi, logError })
   )
 
+  router.use('/offenders/:offenderNo/adjudications', adjudicationsHistoryRouter(elite2Api, logError))
```

Comment:
Most prisoner stuff is now at /prisoner/.  Do we plan going to change this to make it consistent?

Reply:
Small thing which threw me when doing a find was we've got both adjudicationsHistoryRouter and adjudicationHistoryRouter (no s).